core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.html

```diff
@@ -80,11 +80,35 @@ <h4 class="modal-title">Execution results of Workflow {{workflowName}}</h4>
             nzTooltipPlacement="top"
           >
             {{column}}
-            <span *ngIf="column !== '' && column !== 'Status'">
-              <button nz-button (click)="ascSort(column)">
+            <span *ngIf="column !== '' && column == 'Name'">
+              <button *ngIf="this.showORhide[0]" nz-button (click)="onHit(column, 0)">
                 <i nz-icon nzTheme="outline" nzType="sort-ascending"></i>
               </button>
-              <button nz-button (click)="dscSort(column)">
+              <button *ngIf="!this.showORhide[0]" nz-button (click)="onHit(column, 0)">
+                <i nz-icon nzTheme="outline" nzType="sort-descending"></i>
+              </button>
+            </span>
+            <span *ngIf="column !== '' && column == 'Username'">
+              <button *ngIf="this.showORhide[1]" nz-button (click)="onHit(column, 1)">
```

Comment:
why do you need a hardcoded index for each? can't you just toggle it using a boolean variable?

Reply:
because I need to avoid conflicts between sorting icons, I need to hardcode index for each column